src/main/java/edu/greenblitz/robotName/commands/PanicMode.java

```diff
@@ -0,0 +1,19 @@
+package edu.greenblitz.robotName.commands;
+
+import edu.greenblitz.robotName.commands.shooter.MoveShooterToAngle;
+import edu.greenblitz.robotName.commands.arm.MoveElbowAndWristToSafe;
+import edu.greenblitz.robotName.commands.shooter.pivot.MovePivotToAngle;
+import edu.greenblitz.robotName.subsystems.arm.Elbow;
+import edu.greenblitz.robotName.subsystems.shooter.Pivot.PivotConstants;
+import edu.wpi.first.wpilibj2.command.ConditionalCommand;
+
+public class PanicMode extends ConditionalCommand {
+
+    public PanicMode() {
+        super(
+                MoveShooterToAngle.getCommand(PivotConstants.PresetPositions.SAFE.ANGLE),
```

Comment:
if it gets a Rotation2D from a Position consider making the function get a possition

Reply:
no because you would want to move the shooter to any angle and not just preset positions